config.json

```diff
@@ -198,7 +198,7 @@
     "TheDream-Eaters.a34f34",
     "ChaosBag.fea079",
     "DataHelper.708279",
-    "BlessCurseTokenManager.5933fb",
```

Comment:
Does this just replace the current manager?  The code for it is at src/tokens/BlessCurseManager.ttslua, please update that